docs/bugs.rst

```diff
@@ -33,6 +33,8 @@ contracts should consult this list according to the following criteria:
 The JSON file of known bugs below is an array of objects, one for each bug,
 with the following keys:
 
+uid
+    Unique identifier given to the bug in the form of ``SOL-<year>-<number>``
```

Comment:
Maybe we should clarify that you can have two different names with same uid?

Reply:
```suggestion
    Unique identifier given to the bug in the form of ``SOL-<year>-<number>``.
    It is possible that multiple entries exists with the same uid. This means
    multiple version ranges are affected by the same bug.
```

Reply:
Sounds good to me.